server/src/config/config.js

```diff
@@ -21,3 +21,8 @@ export default {
     use_env_variable: 'DATABASE_URL'
   }
 };
+let conString;
```

Comment:
What is a conString ?

Reply:
conString is used as the temporary store for connectionString before exporting connectionString

Reply:
The point is I don't think this is an appropriate name for what it does from your explanation